### DISCUSSION TITLE
What is the "Register as Shell extension" option ?

### DISCUSSION THREAD

**Discussion by mustafatoktas** [valinet/ExplorerPatcher#3292] [Q&A] [answered]
What is the explanation of this setting? What does this setting accomplish when I turn this on? What are the advantages and disadvantages?

![1](https://github.com/valinet/ExplorerPatcher/assets/52534609/da635048-b974-48fd-9185-8097d33f01b3)

**Comment by pyrates999** [comment, marked as answer]
https://github.com/valinet/ExplorerPatcher/wiki/Using-ExplorerPatcher-as-shell-extension
